end-to-end-tests/specs/mutationMapperTool.spec.js

```diff
@@ -14,7 +14,7 @@ describe('Mutation Mapper Tool', function() {
 
     describe('example genomic changes input', ()=>{
         beforeEach(()=>{
-            var url = `${CBIOPORTAL_URL}/mutation_mapper.jsp`;
+            var url = `${CBIOPORTAL_xURL}/s/mutation_mapper`;
```

Comment:
@alisman is this the right way to fix this route?

Reply:
is there a typo CBIOPORTAL_*x*URL

Reply:
oops, yes :)